### PR TITLE
A30: New york times 

### PR DESCRIPTION
**Pull request by Brandonz07**

### CODE REVIEW

File: src/br/html/A30-New-York-Times/styles/style.css [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+    color: white;
+    background-color: black;
+    border: none;
+    border-radius: 2px;
+    padding: 7px 14px;
+    font-size: 0.75em;
+    margin-right: 2%;
+}
+
+article:first-child h2{
```
Comment (carin91) [inline src/br/html/A30-New-York-Times/styles/style.css:156]: you have a class; however, you are styling based on the pseudoelement. I would delete the class define for this one.

File: src/br/html/A30-New-York-Times/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+.news > h3{
+    margin-bottom: 4%;
+}
+
+article > div:nth-child(4){
+    padding-top: 2em;
+    display: flex;
+}
+
+.point:after{
```
Comment (carin91) [inline src/br/html/A30-New-York-Times/styles/style.css:188]: I don't get it. Are you defining a div to render it's after pseudo element? I would condition the div to look like a point or something else, there are too many html nodes without use.

File: src/br/html/A30-New-York-Times/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+.news:nth-child(3){
+    padding-top: 8%;  
+}
+
+.foward > img{
+    width: 35%;
+    padding-left: 6%;
+}
+
+.foward p:last-child{
```
Comment (carin91) [inline src/br/html/A30-New-York-Times/styles/style.css:215]: It is better to define a better html tag for the first paragraph, considering it as a heading.

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/br/html/A30-New-York-Times/A30-New-York-Times.html [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+        <img src="./assests/logo.png" alt="New york times logo">
+        <div>
+            <button>SIGN IN</button>
+            <button>SUSCRIBE</button>
+        </div>
+    </header>
+    <nav>
+        <div>
+            <p>Thursday</p>
+            <p>May 21, 2020</p>
```
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/br/html/A30-New-York-Times/A30-New-York-Times.html]: changed tag

File: src/br/html/A30-New-York-Times/A30-New-York-Times.html [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+            <article>
+                <img src="./assests/article1.png" alt="A frustrated person, on a yellow background">
+                <div class="news">
+                    <div class="detail_and_date">
+                        <span>TECH FIX</span>
+                        <p>May 20, 2020</p>
+                    </div>
+                    <h2>Everything You Need to Know About Slow Internet Speeds</h2>
+                    <p>Our Crummy connections are the biggest tech headache in the pandemic. Here's a comprehensive guide to do about them.</p>
+                    <button class="continue">Continue Reading</button>
```
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/br/html/A30-New-York-Times/A30-New-York-Times.html]: changed tag a

File: src/br/html/A30-New-York-Times/A30-New-York-Times.html [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+                </div>
+            </article>
+            <article>
+                <h3>Lastest News</h3>
+                <div class="news">
```
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/br/html/A30-New-York-Times/A30-New-York-Times.html:85]: very similar styles

File: src/br/html/A30-New-York-Times/A30-New-York-Times.html [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+                </div>
+                <div class="news">
+                    <div class="detail_and_date">
+                        <span>NEW ANALYSIS</span>
+                        <p>May 19, 2020</p>
+                    </div>
+                    <h2>Coronavirus Shut Down the 'Experience Economy.' Can It Come Back?</h2>
+                    <button class="continue">Continue Reading</button>
+                </div>
+                <div id="ponts">
```
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/br/html/A30-New-York-Times/A30-New-York-Times.html]: delete id

File: src/br/html/A30-New-York-Times/A30-New-York-Times.html [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+                    <div class="point"></div>
+                    <div class="point"></div>
+                    <div class="point"></div>
+                </div>
+            </article>
+            <article>
+                <h3>Fast Foward</h3>
+                <div class="foward">
```
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/br/html/A30-New-York-Times/A30-New-York-Times.html]: changed name

File: src/br/html/A30-New-York-Times/A30-New-York-Times.html [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+                    <div>
+                        <p>Another Grim Tally of U.S. Unemployement Is Exoected: Live Business Updates</p>
+                        <p>Live stock Market News Darling the Coronavirus Pandemic</p>
+                    </div>
+                    <img src="./assests/fast1.png" alt="abandoned streets of New York">
+                </div>
+                <div class="foward">
+                    <div>
+                        <p>Another Grim Tally of U.S. Unemployement Is Exoected: Live Business Updates</p>
```
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/br/html/A30-New-York-Times/A30-New-York-Times.html]: i use h3

File: src/br/html/A30-New-York-Times/styles/style.css [hunk 10]
```diff
@@ -0,0 +1,256 @@
...
+
+body{
+    width: 90%;
+    margin: 0 auto;
+    background-color: #F7F7F7;
+    margin-left: 6.5%;
+}
+
+ul:first-child{
+    margin-left: -6.5%;
```
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/br/html/A30-New-York-Times/styles/style.css]: I don't use negatives anymore

File: src/br/html/A30-New-York-Times/styles/style.css [hunk 11]
```diff
@@ -0,0 +1,256 @@
...
+    position: fixed;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    background-color: white;
+    width: 3%;
```
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/br/html/A30-New-York-Times/styles/style.css]: i use fixed-size units

File: src/br/html/A30-New-York-Times/styles/style.css [hunk 12]
```diff
@@ -0,0 +1,256 @@
...
+    margin: 0 auto;
+    background-color: #F7F7F7;
+    margin-left: 6.5%;
+}
+
+ul:first-child{
+    margin-left: -6.5%;
+    position: fixed;
+    height: 100%;
+    display: flex;
```
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/br/html/A30-New-York-Times/styles/style.css]: no

File: src/br/html/A30-New-York-Times/styles/style.css [hunk 13]
```diff
@@ -0,0 +1,256 @@
...
+    flex-direction: column;
+    background-color: white;
+    width: 3%;
+    align-items: center;
+    justify-content: space-between;
+}
+
+ul:first-child > li > a > i{
```
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/br/html/A30-New-York-Times/styles/style.css]: changed selectors

File: src/br/html/A30-New-York-Times/styles/style.css [hunk 14]
```diff
@@ -0,0 +1,256 @@
...
+
+header > div > i:nth-child(2)::after{
+    content: "●";
+    color: transparent;
+    font-size: 50%;
+    background-color: red;
+    border: 0.5px solid white; 
+    border-radius: 50%;
+    position: relative;
+    margin-left: -25%;
```
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/br/html/A30-New-York-Times/styles/style.css]: i use left

File: src/br/html/A30-New-York-Times/styles/style.css [hunk 15]
```diff
@@ -0,0 +1,256 @@
...
+    top: -0.8em;
+}
+
+header button{
+    font-size: 1em;
+    font-weight: bold;
+    padding: 8px 15px;
+    background-color: transparent;
+    border: 1px solid black;
```
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/br/html/A30-New-York-Times/styles/style.css:61]: add a hover

File: src/br/html/A30-New-York-Times/styles/style.css [hunk 16]
```diff
@@ -0,0 +1,256 @@
...
+
+nav:nth-child(3) > ul{
+    list-style: none;
+    margin: 0 2%;
+    width: 100%;
+    justify-content: space-between;
+}
+
+a{
+    text-decoration: none;
```
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/br/html/A30-New-York-Times/styles/style.css:112]: add a underline

File: src/br/html/A30-New-York-Times/styles/style.css [hunk 17]
```diff
@@ -0,0 +1,256 @@
...
+    color: rgb(167, 166, 166);
```
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/br/html/A30-New-York-Times/styles/style.css]: changed font and color

File: src/br/html/A30-New-York-Times/styles/style.css (hunk 2 above)
Comment (Brandonz07) [reply src/br/html/A30-New-York-Times/styles/style.css]: don't use pseudo elements

**Comment by Brandonz07** [comment]
I used grid instead of flex, add the nav to the header, change the section to main, and in the last column use float

**Review by carin91 — APPROVED** [review]

**Pull request merged**